test/run_test.py

```diff
@@ -198,7 +198,7 @@ def skip_test_p(name: str) -> bool:
     "distributed/elastic/multiprocessing/api_test",
     "distributed/_sharded_tensor/test_sharded_tensor",
     "distributed/_sharded_tensor/ops/test_linear",
-]
+] + [test for test in TESTS if test.startswith("distributed/fsdp")]
```

Comment:
nit: since we do this 3 times, do we want to have a FSDP_TEST variable that hold this?

Also the reason to use this even though there is a single file is for future proofing?